tests/Integration/Behaviour/Features/Scenario/Product/update_pack.feature

```diff
@@ -176,11 +176,11 @@ Feature: Add product to pack from Back Office (BO)
       | productSkirt1 | productSkirt1MWhite | 11       |
       | productSkirt1 | productSkirt1MBlack | 12       |
     Then product "productPack4" type should be pack
-    And pack productPack4 should contain products with following quantities:
-      | product       | combination         | quantity |
-      | productSkirt1 | productSkirt1SWhite | 10       |
-      | productSkirt1 | productSkirt1MWhite | 11       |
-      | productSkirt1 | productSkirt1MBlack | 12       |
+    And pack productPack4 should contain products with following details:
+      | product       | name                                    | combination         | quantity | image url                                              |
+      | productSkirt1 | regular skirt - Size - S, Color - White | productSkirt1SWhite | 10       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
```

Comment:
```suggestion
      | productSkirt1 | regular skirt: Size - S, Color - White | productSkirt1SWhite | 10       | http://myshop.com/img/p/{no_picture}-small_default.jpg |
```

Reply:
I agree for the `,` separator between attributes and the ` - ` separator between attribute group and their value However it feels weird that ` - ` is used after the product name Is it how it was displayed in the legacy page already? If so then ignore this comment

Reply:
Code change to provide line as :
regular skirt : Size - M, Color - Black
Test also refactored inthsi way